docs/pages/management/dynamic-resources/user-and-role.mdx

```diff
@@ -0,0 +1,501 @@
+---
+title: Managing users and roles with IaC
+description: Use Infrastructure-as-Code tooling to create Teleport users and roles.
+---
+
+In this guide, you will see how to create users and grant them roles through
+Infrastructure-as-Code (IaC). Teleports supports three ways to dynamically
+create resources from code:
+
+- The Teleport Kubernetes Operator that allows you to manage Teleport resources
+  from Kubernetes
+- The Teleport Terraform Provider that allows you to manage Teleport Resources
+  via Terraform
+- The `tctl` CLI that allows you to manage Teleport resources from your local
+  computer or your CI environment
+
+### Prerequisites
+
+To follow this guide, you must have:
+
+<Tabs>
+<TabItem label="tctl">
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
```

Comment:
Instead of this partial, I would include instructions to download `tctl`.

Also, if we are going to include adding a Teleport cluster as a prerequisite, we should add it for tctl, the Operator, and the Terraform provider.

Reply:
For the operator and the terraform cluster, the Teleport cluster is a requirement in their respective guides.
If you have a working operator/TF provider, you have a running Teleport cluster somewhere.

The operator and terraform requirements are a bit specific wrt to the kind of teleport cluster they can connect to, so I think it's better to leave to teleport requirement in the dedicated guide instead of putting an awkward generic requirement here.